docs/Documentation/Development/Libraries/hsl.md

```diff
@@ -8,19 +8,19 @@ parent: Libraries
 grand_parent: Development
 ---
 
-# HSL for Ipopt
+# HSL for IPOPT
 
-*HSL (Harwell Subroutine Library) for Ipopt are a set of linear solvers that can greatly accelerate the speed of the optimization over the default MUMPS solver.*
+*HSL (Harwell Subroutine Library) for IPOPT are a set of linear solvers that can greatly accelerate the speed of the optimization over the default MUMPS solver.*
 
 ## Installation
 
-Go to the [HSL for Ipopt](http://www.hsl.rl.ac.uk/ipopt/) site and follow the instructions to request the source code for all the available solvers. Note that the solver MA27 is free to obtain, but MA27 is a serial solver. Other solvers will require a license. Please request a license that applies to your use case.
+Go to the [HSL for IPOPT](http://www.hsl.rl.ac.uk/IPOPT/) site and follow the instructions to request the source code for all the available solvers. Note that the solver MA27 is free to obtain, but MA27 is a serial solver. Other solvers will require a license. Please request a license that applies to your use case.
 
-### Eagle
+### HPC
```

Comment:
I don't think we need the HPC header here, since the documentation site is for HPC. Can remove and move the other section headings up a level

Reply:
So the above is to differentiate from the instructions for installing on Mac below (line 88).